libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -259,6 +299,15 @@ export class TokenizerComponent implements AfterViewChecked, AfterViewInit, Afte
             this.previousElementWidth = elementWidth;
         }
     }
+    /** @hidden */
+    handleKeyUp(event: KeyboardEvent): void {
```

Comment:
I believe you can use `KeyboardEvent.ctrlKey | .shiftKey` to determine if someone is combining keys.

Reply:
So the thing is that it is not for when they combine keys
Holding shift and control do two different things
And depending on what they are holding will hit the proper if statement